core/quantity/src/Parser.ts

```diff
@@ -796,6 +800,62 @@ export class Parser {
     return { ok: true, value: magnitude };
   }
 
+  private static parseRatioFormat(inString: string, spec: ParserSpec): QuantityParseResult {
+    if (!inString)
+      return { ok: false, error: ParseError.NoValueOrUnitFoundInString };
+
+    const parts = inString.split(":");
+    if (parts.length > 2)
+      return { ok: false, error: ParseError.UnableToConvertParseTokensToQuantity };
+
+    const numerator = parseFloat(parts[0]);
+    let denominator;
+    if (parts.length === 1) {
+      denominator = 1.0;
+    } else {
+      denominator = parseFloat(parts[1]);
+    }
+
+    if (isNaN(numerator) || isNaN(denominator))
+      return { ok: false, error: ParseError.NoValueOrUnitFoundInString };
+
+    const defaultUnit = spec.format.units && spec.format.units.length > 0 ? spec.format.units[0][0] : undefined;
+    const unitConversion = defaultUnit ? Parser.tryFindUnitConversion(defaultUnit.label, spec.unitConversions, defaultUnit) : undefined;
+
+    if (!unitConversion) {
+      throw new QuantityError(QuantityStatus.MissingRequiredProperty, `Missing input unit or unit conversion for interpreting ${spec.format.name}.`);
+    }
+
+    if (denominator === 0){
+      if (unitConversion.inversion && numerator === 1)
+        return { ok: true, value: 0.0 };
+      else
+        return { ok: false, error: ParseError.MathematicOperationFoundButIsNotAllowed };
+    }
+
+    let quantity: Quantity;
+    if (spec.format.units && spec.outUnit) {
+      quantity = new Quantity(spec.format.units[0][0], numerator / denominator);
+    } else {
+      throw new QuantityError(QuantityStatus.MissingRequiredProperty, "Missing presentation unit or persistence unit for ratio format.");
+    }
+
+    let converted: Quantity | undefined;
+    try {
+      converted = quantity.convertTo(spec.outUnit, unitConversion);
+    } catch (err){
+      if (err instanceof QuantityError && err.errorNumber === QuantityStatus.InvertingZero){
```

Comment:
I think you caught this further up. I don't see any test hitting this code, and I think it's probably wrong. The result of a zero inversion should not be zero.